docs: Drop single user support (#2103)
codenvy/che-docs: modules/administration-guide/partials/con_che-workspace-controller.adoc

[id="{prod-id-short}-workspace-controller_{context}"]
= {prod-short} workspace controller

> The workspaces controller manages the container-based development environments: {prod-short} workspaces. Following deployment scenario is available:

Reviewer: ```suggestion
The workspaces controller manages the container-based development environments: {prod-short} workspaces. To secure the development environements with authentication, the deployment is always multiuser and multitenant.
```

Simplify, as we have only one scenario, the list becomes unnecessary.

> * *Multiuser*: This is a multitenant configuration. Development environments are secured. Appropriate for cloud installations.

Reviewer: ```suggestion
```

Remove, concatenate with previous paragraph.

[id="{prod-id-short}-workspace-controller_{context}"]
= {prod-short} workspace controller

> The workspaces controller manages the container-based development environments: {prod-short} workspaces. To secure the development environements with authentication, the deployment is always multiuser and multitenant.

Reviewer: ```suggestion
The workspaces controller manages the container-based development environments: {prod-short} workspaces. To secure the development environments with authentication, the deployment is always multiuser and multitenant.
```
